### ISSUE TITLE
v1.0.5.1 - Filter does not work under Linux

### ISSUE THREAD

**Issue by GoogleCodeExporter** [keithgg/puddletag#253]
```
Fedora 21 - all packages updated to latest versions of distribution.

After updating to v1.0.5.1,

Filter does not work. No effect, entering a string an pressing GO.

Downgraded to v1.0.3.1 and filter worked back properly.

Other functions seem to work.
Could not see any output in terminal (even in debug mode).

Please, tell me how I can help debugging this issue!


```

Original issue reported on code.google.com by `alexhfra...@gmail.com` on 22 Feb 2015 at 9:08

**Comment by endlisnis** [comment]
I experience the same issue with version 1.0.5 on Linux Mint 17.2.  The filter doesn't seem to do anything.

**Comment by careca1970** [comment]
Hello! Good to now that other people have same problem.
I feel, my issue haven't been taken much seriously here...

Do you have an idea, how we can collect debug information on this?

**Comment by careca1970** [comment]
Hi! I want to correct this entry.
In fact, searches with FIELDs or OPERATORs do work.

Only searches with a single "STRING" does not work at:

Puddletag v1.0.5.1
Fedora 22 (fully updated)

I hope we can try fault finding the issue now, with more accurate info.

**Comment by keithgg** [comment]
Fixed in 21e1e39a10a956a56b18d40492f573a1799edbfc
